orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/purple/PurpleGeneCopyNumber.java

```diff
@@ -0,0 +1,28 @@
+package com.hartwig.hmftools.datamodel.purple;
+
+import org.immutables.gson.Gson;
+import org.immutables.value.Value;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Gson.TypeAdapters
+@Value.Immutable
+@Value.Style(passAnnotations = {NotNull.class, Nullable.class})
+public interface PurpleGeneCopyNumber {
+
+    @NotNull
+    String chromosome();
+
+    @NotNull
+    String chromosomeBand();
+
+    @NotNull
+    String geneName();
+
+    @Nullable
+    Double minCopyNumber();
+
+    @Nullable
+    Double minMinorAlleleCopyNumber();
```

Comment:
Why are these fields @Nullable?

Reply:
Taken over from oncoact. Seems to be changed by Lieke in January. Judging from `GeneCopyNumber` in common, these fields should not be nullable.